SPECS/json-c/json-c.spec

```diff
@@ -1,9 +1,9 @@
-%global publishdate 20200726
+%global publishdate 20230812
```

Comment:
Can we please attach a test build and fill the pr checklist.

Reply:
please see the updated pr description. Thanks